Translation corrections (#93)
07th-mod/higurashi-console-arcs: Update/some_007.txt

//r一瞬、怒りに燃え上がった目できっ！　と鋭く睨まれて思わず身をすくめたが、私を見て正気を取り戻してくれたのか、山瀬さんは表情を緩めてその手をゆっくりと離した。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "一瞬、怒りに燃え上がった目できっ！　と鋭く睨まれて思わず身をすくめたが、私を見て正気を取り戻してくれたのか、山瀬さんは表情を緩めてその手をゆっくりと離した。",

> 			NULL, "She instictively tensed up and glared my way with rage burning in her eyes, but then her face relaxed and she slowly let go,, as if looking at me had brought her back to her senses.", Line_Normal);

Reviewer: ```suggestion
			NULL, "She instictively tensed up and glared my way with burning rage in her eyes, but then her face relaxed and she slowly let go, as if looking at me had brought her back to her senses.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>山瀬</color>", NULL, "<color=#f5e6d3>Yamase</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamase1036", 256, TRUE);
	OutputLine(NULL, "「南井、",
			NULL, "\"Detective,", Line_WaitForInput);
	ModPlayVoiceLS(4, 0, "ps3/s04/00/vt1b_yamase1037", 256, TRUE);
	OutputLine(NULL, "——刑事っ…？！」",
			NULL, " ...Minai...?!\"", Line_Normal);
	ClearMessage();

//rさっ、と山瀬さんの表情に、緊張が走る。そして、私もきっと同じだと思う。少なくとも身に覚えのないにもかかわらず『ご協力』と警察の人に言われて、嫌な想像をしないわけがない。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "さっ、と山瀬さんの表情に、緊張が走る。そして、私もきっと同じだと思う。少なくとも身に覚えのないにもかかわらず『ご協力』と警察の人に言われて、嫌な想像をしないわけがない。",

> 			NULL, "Yamase-san's face tensed up, and I was sure I was no different. At the very least, I couldn't imagine anything good coming from the police ask for your <i>cooperation</i> when you hadn't done anything.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Yamase-san's face tensed up, and I was sure I was no different. At the very least, I couldn't imagine anything good coming from the police asking for your <i>cooperation</i> when you hadn't done anything.", Line_Normal);
```

	ClearMessage();

//rそして、その人も慣れているのか、別に媚びるのでも高圧的になるのでもなく、自然体で手帳を見せる。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、その人も慣れているのか、別に媚びるのでも高圧的になるのでもなく、自然体で手帳を見せる。",

> 			NULL, "This woman seemed used to that, as she took out her notebook without sucking up to us or getting high-handed.", Line_Normal);

Reviewer: ```suggestion
			NULL, "This woman seemed used to that, as she calmly took out her notebook without resorting to flattery or acting overbearing.", Line_Normal);
```

//rぶるっ、と身体に震えを感じずにはいられなかった。この南井刑事、あらかじめ反応を想定して、ちゃんと対策を打ってきている…？！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "ぶるっ、と身体に震えを感じずにはいられなかった。この南井刑事、あらかじめ反応を想定して、ちゃんと対策を打ってきている…？！",

> 			NULL, "I couldn't help but feel my body trembling. This Detective Minai had anticipated our reactions in advance and took proper countermeasures...?!", Line_Normal);

Reviewer: ```suggestion
			NULL, "I couldn't help but feel my body trembling. This Detective Minai had anticipated our reactions in advance and prepared suitable countermeasures...?!", Line_Normal);
```

//rそして、しゃがんで俯きながらブツブツと呟き続ける畠山のおじーちゃんの目線に合わせるように、その場に膝をついて顔をのぞき込んで言った。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、しゃがんで俯きながらブツブツと呟き続ける畠山のおじーちゃんの目線に合わせるように、その場に膝をついて顔をのぞき込んで言った。",

> 			NULL, "Then, she crouched down to look the mumbling old man Hatakeyama, lying prone on the floor, in the eyes.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Then, she crouched down to look the mumbling old man Hatakeyama in the eyes as he lay prone on the floor.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 28, "ps3/s04/28/310500138", 256, TRUE);
	OutputLine(NULL, "「これが撮られた今年の６月１９日、あなたもここのお祭りに参加して、そしてこの少女と会っている。",

> 			NULL, "\"On June 19th of this year, the date of this photo, you attended the festival and met this girl.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"On June 19th of this year, the date this photo was taken, you attended the festival and met this girl.", Line_WaitForInput);
```

//rまして、…雛見沢から離れて暮らしている畠山のおじーちゃんに、一体何の関係があるの？　そんなこと言いだしたら、私だって…。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "まして、…雛見沢から離れて暮らしている畠山のおじーちゃんに、一体何の関係があるの？　そんなこと言いだしたら、私だって…。",

> 			NULL, "Also... what was the connection to old man Hatakeyama, who had been living away from Hinamizawa? I wanted to ask that, but if I did, then I'd end up in the same boat as him...", Line_Normal);

Reviewer: ```suggestion
			NULL, "Also... what was the connection to old man Hatakeyama, who had been living away from Hinamizawa? I wanted to ask, but if I did, then I'd end up in the same boat as him...", Line_Normal);
```
